### PR TITLE
Add core-docs project

### PR DESCRIPTION
**Pull request by cartermp**
We've got some C# tutorials content that's perfect for community members to contribute to.

### CODE REVIEW

File: _data/projects/dotnet-core-docs.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: .NET Core Docs
+desc: Documentation for .NET Core
+site: http://github.com/dotnet/core-docs
+tags:
+- ".NET"
```
Comment (shiftkey) [inline _data/projects/dotnet-core-docs.yml:5]: Add a `documentation` tag in here so it's easier to find

Comment (cartermp) [inline _data/projects/dotnet-core-docs.yml:5]: Will do

**Comment by shiftkey** [comment]
Thanks!

**Comment by cartermp** [comment]
:+1: Thank you too!

**Pull request merged**